init-sensor.h

```diff
@@ -42,6 +42,9 @@ void initSensorType() {
     case SENSOR_DS18B20:
       initDallasSensor();
       break;
+    case SENSOR_ANALOG_TEST:
```

Comment:
should we do that only for potentiometer? (SENSOR_ANALOG_TEST is potentiometer)?
or should we do that for GUT800 and INA as well?
or should we do that for GUT800 and INA in another way?

Reply:
I guess that this should be done for GUT800 and INA only.
For potentiometer we need to measure whole range(0-3.3V).

Reply:
@shybzzz In which cases we are using potentiometer solution? What is the purpose of this device?